python/lsst/ctrl/bps/cancel.py

```diff
@@ -0,0 +1,66 @@
+# This file is part of ctrl_bps.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""API for cancelling runs submitted to a WMS.
+"""
+
+import logging
+
+from lsst.utils import doImport
+
+_LOG = logging.getLogger()
+
+
+def cancel(wms_service, wms_id, user, require_bps, pass_thru):
+    """Cancel submitted workflows.
+
+    Parameters
+    ----------
+    wms_service : `str` or `~lsst.ctrl.bps.wms_service.WmsService`
+        Name of the Workload Management System service class.
+    user : `str`
+        Cancel all submitted workflows for specified user.
+    require_bps : `bool`
+        Require given run_id/user to be a bps submitted job.
+    wms_id : `str`
+        Cancel submitted jobs matching specified WMS id.
+    pass_thru : `str`
+        A string to pass directly to the WMS service class.
```

Comment:
Can we list parameters in the description in the order matching their order in function's signature?